model.py

```diff
@@ -0,0 +1,46 @@
+import re
+
+
+def show_dict():
+    with open('phone.txt', 'r') as file:
+        print(file.read())
+
+
+def enter_cont():
+    id = input('Введите id контакта: ')
+    first_name = input('Введите имя: ')
+    last_name = input('Введите фамилию: ')
+    phone = input('Введите номер телефона: ')
+    birthday = input('Укажите дату рождения: ')
+    work = input('Укажите место работы: ')
+    phone_book = f'\n{id} {first_name} {last_name} {phone} {birthday} {work}'
+    with open('phone.txt', 'a+', encoding='utf-8') as file:
+        file.write(phone_book)
+    print(f'Добавлен новый контакт:{phone_book}')
+
+
+def find_cont():
+    value = input('Введите значение для поиска: ')
+    with open(r'phone.txt', 'r') as file:
+        lines = file.readlines()
+        for line in lines:
+            if line.find(value) != -1:
```

Comment:
Эта строчка не обязательна, потому что объект `file` итерируемый, и по нему можно пройтись просто циклом.
```suggestion
        for line in file:
            if line.find(value) != -1:
```